### PR TITLE
update event & quote for 7 aug

### PR DESCRIPTION
**Pull request by ando-hunterz**

### CODE REVIEW

File: content/english/events/20210807.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        ]
+tags: ["Fellowship"]
+math: false
+toc: true
+draft: false
+---
+
+![image](/images/events/20210807.jpeg)
+
+Hello, this saturday, our youth fellowship will held a seminar about Pop Culture.
```
Comment (jwangsadinata) [inline content/english/events/20210807.md:24]: capitalize some words, in particular change it to this:

`Hello, this Saturday, our Youth Fellowship group will have a Pop Culture seminar. Come and join.

**Review by jwangsadinata — CHANGES_REQUESTED** [review]
@ando-hunterz please fix the English version content.

File: content/english/events/20210807.md (hunk 1 above)
Comment (ando-hunterz) [reply content/english/events/20210807.md]: done

**Review by jwangsadinata — APPROVED** [review]
LGTM.

**Pull request merged**